exercises/concept/black-jack/.docs/instructions.md

```diff
@@ -1,49 +1,85 @@
-In this exercise, you're going to implement some rules from [Blackjack][blackjack]
+# Instructions
 
-You have some rules to implement for judging the result of the game.
+In this exercise you are going to implement some rules of [Blackjack][blackjack],
+such as the way the game is played and scored.
 
-**Note** : In this exercise, _A_ means an ace, _J_ means jack, _Q_ means queen, and _K_ means king card.
+**Note** : In this exercise, _A_ means ace, _J_ means jack, _Q_ means queen, and _K_ means king cards.
 
-## 1. Calculate the number of card
+## 1. Calculate the value of a card
 
-Create the `number_of_card()` function with a parameter `card`. The value of _J, Q_ or _K_ is 10. If the `card` is _A_, then just return "ace".
+In Blackjack, it is up to each individual player if an ace is worth 1 or 11 points.
+Face cards are worth 10 points and any other card is worth its pip value.
```

Comment:
I am fine with the language as it stands, but am wondering if "face card" is universally (more or less) understood?  If not, we should put a link in to clarify.

Reply:
I've now put the corresponding cards in parentheses.
Also, added a standard 52-card wiki link at the beginning of the instructions.